Main/styles/common/_tweet-dialog.scss

```diff
@@ -4,61 +4,71 @@
     left: 0;
     width: 100%;
     height: 100%;
-
     display: flex;
     align-items: center;
     justify-content: center;
-
     z-index: 400;
     background-color: #00000080;
+}
 
-    .tweet-dialog {
-        background-color: white;
-        border-radius: 4px;
-        box-shadow: 0px 2px 10px 0px #0000001A;
-        padding: 8px 16px;
-
-        display: flex;
-        flex-direction: column;
-        gap: 10px;
-
-        h1 {
-            font-family: Pretendard;
-            font-size: 18px;
-            font-weight: 700;
-            line-height: 21px;
-            letter-spacing: -0.01em;
-            text-align: left;
-        }
-
-        textarea {
-            font-family: Pretendard;
-            font-size: 14px;
-            font-weight: 400;
-            line-height: 17px;
-            letter-spacing: -0.01em;
-            text-align: left;
+.tweet-dialog {
```

Comment:
Nest .tweet-dialog into .tweet-dialog-wrapper.

For example, Write like A, not B.

```scss
// A
.tweet-dialog-wrapper {
   // Do something
   .tweet-dialog {
      // Do something
   }
}
```

```scss
// B
.tweet-dialog-wrapper {
   // Do something
}
.tweet-dialog {
   // Do something
}
```